StockWatch.Backend/src/main/java/com/henning/oden/java/StockAlert/Backend/util/dev/DataSeeder.java

```diff
@@ -0,0 +1,49 @@
+// TODO: Overhaul or remove this implementation of concepts presented by SeunMatt in https://github.com/SeunMatt/spring-blog/blob/master/src/main/java/com/smatt/seeders/DatabaseSeeder.java
+package com.henning.oden.java.StockAlert.Backend.util.dev;
+
+import com.henning.oden.java.StockAlert.Backend.entities.Stock;
+import com.henning.oden.java.StockAlert.Backend.entities.SystemUser;
+import com.henning.oden.java.StockAlert.Backend.repos.StockRepository;
+import com.henning.oden.java.StockAlert.Backend.repos.SystemUserRepository;
+import liquibase.pro.packaged.S;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+@Component
+public class DataSeeder {
```

Comment:
DO it in liquibase and remove this class

Reply:
Attempt to do this with Liquibase fails as the password is encrypted with BCrypt and login seems to become impossible after seeding with Liquibase.

Reply:
Not sure I understand the probl. 

You encrypt, and use the result  string as value, insert ok but then when compares on login doesn't work?

Reply:
Yes. I might have missed a letter somewhere, so I am going to try again

Reply:
I have checked to make sure I didn't miss any characters and for some reason, login just fails with the inserted password hash. Doing everything over from the start with the SeedData class implementation works, not the Liquibase approach. I have confirmed that the entry in the database is the same in both cases.